### DISCUSSION TITLE
Suggested simplification of Armament construction and staffing.

### DISCUSSION THREAD

**Discussion by sum1els-com** [jacobsalmela/ascii-planets#39] [Ideas]
Currently, an Armament may be constructed if the player has no available Astronauts, provided that the player staffs the Armament as soon as an Astronaut becomes available.  When the Armament is staffed, the appropriate letter is written into the box to indicate what kind of Armament it is.

This rule is not only a bit complicated, but can also lead to some confusing or ambiguous states.  For example, if I have no available Astronauts I could still theoretically construct an Academy, Factory, and Lab all on the same turn.  Next turn I have three empty boxes, but there is no way to tell which one is which or even whether they are all the same.

The simplest way to both streamline the rules and avoid the above complication is to simply require that Armaments are staffed when they are built, meaning that you simply cannot build an Armament if you have no available Astronauts.  With the ability to purchase Astronauts using Currency, the cases where this simplification ruins someone's day should be rare.

**Comment by jacobsalmela** [comment]
That makes good sense.  I appreciate you picking apart the game.  

For context, I wanted players to be able to build and build and then staff, but you're right, you cannot tell them apart.  In Empires, these were different shaped buildings, but with the focus more on outer space, I didn't want the players trying to fit shapes into the round planet.  

Part of it the problem is physical space on the sheet.  Making the game in LaTeX has been great for some things but positioning elements on the sheet is not one of them.  It's easier with TikZ nodes, which I am still working on over here.

Maybe increasing the costs of some things may help, or maybe removing a staffing rule all together in favor of just building things to get to outer space faster.

**Reply by sum1els-com** [reply]
I totally get that you wanted to provide the option of staffing later, but in practice I don't think it would really come up much anyway.  Was this a big factor in playtests?  I haven't had the opportunity to try it out with my game group yet.

The main thing keeping people from building Starships is the cost.  It's much more effective to focus on technology first and get the extra dice.  The dynamic might change in a group if others are all building convoys rapidly, which would encourage you to build a Starship early to gain reach.  In the solo game the primary reason to build Starships is to get to the Battleship, which really does need more dice.

**Comment by jacobsalmela** [comment]
https://github.com/jacobsalmela/ascii-planets/issues/41

**Comment by jacobsalmela** [comment]
This should be addressed in https://github.com/jacobsalmela/ascii-planets/releases/tag/v1.2.5. 

Feel free to mark this as an answer for anyone else landing here.
